app/components/public/ticket-list.js

```diff
@@ -33,6 +33,12 @@ export default Component.extend(FormMixin, {
     ) > 0;
   }),
 
+  tax_calculated_price: computed('event', 'price', function() {
+    if (this.get('event').tax.rate) {
```

Comment:
This won't work in a component, reason being we can't access a ticket's price by this.

Reply:
Calculated it using total price.